public/content/documentation/web/component/password-input.md

```diff
@@ -0,0 +1,218 @@
+     - I HEAR Its purpose is clear
+     - I HEAR It identifies itself as a text input
+     - I HEAR Hints or errors are read after the label (Ex: Password formatting)
+     - I HEAR It expresses if the password is being shown and if applicable: required, disabled / dimmed / unavailable
+   - THEN when I use the tab key to move focus to the show/hide password feature I HEAR its name, role and state
+   - THEN when I use the show/hide password feature I HEAR the state of the password visibility (with or without characters entered)
+
+3. Mobile screenreader
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND I swipe to focus on a password input
+     - I HEAR Its purpose is clear
+     - I HEAR It identifies itself as a text input
+     - I HEAR Hints or errors are read after the label (Ex: Password formatting)
+     - I HEAR It expresses if the password is being shown and if applicable: required, disabled / dimmed / unavailable
+
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/password-input](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/password-input)
+
+## Code Examples
+
+### Checkbox variant
+   - This semantic HTML contains all accessibility features by default.
+   - Placing the show password checkbox ahead of the password input increases discoverability for screen reader users.
+     - CSS pseudo elements are used in the checkbox label to express its state on focus of the password input.
+
+```html
+<form class="password-container">
+  <label for="password">
+    Password
+    <span class="helper" aria-hidden="true">
+      Required
+    </span>
+  </label>
+  <div id="password-toggle" class="password-toggle">
+    <input type="checkbox" id="show-password">
+    <label for="show-password">
+      Show Password
+    </label>
+  </div>
+  <input type="password"
+         id="password"
+         aria-describedby="password-toggle password-hint"
+         aria-required="true"
+         required>
+  <div id="password-hint" class="hint">
+    Use any length of characters including emojis.
+  </div>
+</form>
+```
+
+<example>
+<form class="password-container">
+  <label for="password">
+    Password
+    <span class="helper" aria-hidden="true">
+      Required
+    </span>
+  </label>
+  <div id="password-toggle" class="password-toggle">
+    <input type="checkbox" id="show-password">
+    <label for="show-password">
+      Show Password
+    </label>
+  </div>
+  <input type="password"
+         id="password"
+         aria-describedby="password-toggle password-hint"
+         aria-required="true"
+         required>
+  <div id="password-hint" class="hint">
+    Use any length of characters including emojis.
+  </div>
+</form>
+</example>
+
+### Button variant
+   - The button leads the input so screen reader and keyboard-only users can change the state before interacting with the field.
```

Comment:
I know this is what is in current-day MagentaA11Y, but the "button leads the input" but I don't think it is clear that the meaning is that the show/hide button gets focus first, ahead of the input field. How about... "The show/hide button receives focus prior to the input field so that screen reader... (the rest the same as stated)"?